common/cmake/xml.cmake

```diff
@@ -31,11 +31,13 @@ function(XML_LINT)
     --schema ${XML_LINT_SCHEMA}
     ${XML_LINT_FILE} 2>&1 |  head -n10 && exit "$$\{PIPESTATUS[0]\}" '
     )
+
+  get_rel_target(REL_XML_LINT_NAME lint ${XML_LINT_NAME})
```

Comment:
Same comment here.  NAME is no longer the target name, so the parameter is poorly named.  Need to rename the argument and update the function documentation.

Reply:
@kgugala